server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -1000,7 +1000,7 @@ export function configureWebSocketServices(
 								}
 
 								const sanitizedMessage: IDocumentMessage = sanitizeMessage(message);
-								const sanitizedMessageWithTrace = addAlfredTrace(
```

Comment:
What about lambda/src/alfred/index.ts file?
We need to remove that as well.

Reply:
It became the src/nexus/index.ts file